recipes/platform.hashing/all/conanfile.py

```diff
@@ -1,5 +1,5 @@
-from conans import ConanFile, CMake, tools
-from conans.errors import ConanInvalidConfiguration
+from conan import ConanFile, CMake, tools
```

Comment:
```suggestion
from conan import ConanFile
from conan.tools.cmake import CMake
```
and other tools are imported from various namespaces in v2

Reply:
@SSE4 Where can I find python conan package docs? [This one](https://pypi.org/project/conan/) brings me to https://docs.conan.io/en/latest/ . I want to know where should I import `tools` from

Reply:
@uilianries maybe you can help me? :)